### DISCUSSION TITLE
How to add a Popup only to the button block when we click on

### DISCUSSION THREAD

**Discussion by Vishwa-Vxceed** [GrapesJS/grapesjs#5382] [Q&A]
We are trying to open a popup with some select box when we click only on button blocks.. There will be some actions to the buttons.. When we click on the button block a popup needs to be opened and then in the popup just want to show the list of actions in the select box. And then,  when the user selects an action and saves it, just wants to assign that action to the button tag as a data-attribute or class.

Please suggest on how to achieve this in Grapejs..

**Comment by artf** [comment]
That should be possible via a [custom Component view](https://grapesjs.com/docs/modules/Components.html#view)
